src/bindgen/ir/constant.rs

```diff
@@ -160,29 +160,20 @@ impl Literal {
                 lit: syn::Lit::Int(ref value),
                 ..
             }) => match value.suffix() {
-                syn::IntSuffix::Usize
-                | syn::IntSuffix::U8
-                | syn::IntSuffix::U16
-                | syn::IntSuffix::U32
-                | syn::IntSuffix::U64
-                | syn::IntSuffix::U128
-                | syn::IntSuffix::None => Ok(Literal::Expr(format!("{}", value.value()))),
-                syn::IntSuffix::Isize
-                | syn::IntSuffix::I8
-                | syn::IntSuffix::I16
-                | syn::IntSuffix::I32
-                | syn::IntSuffix::I64
-                | syn::IntSuffix::I128 => unsafe {
+                "isize" | "i8" | "i16" | "i32" | "i64" | "i128" => unsafe {
                     Ok(Literal::Expr(format!(
                         "{}",
-                        mem::transmute::<u64, i64>(value.value())
+                        mem::transmute::<u64, i64>(
+                            value.base10_parse::<u64>().expect("Wrong number literal")
```

Comment:
Can we use `base10_parse::<i64>` directly now?

Reply:
Or just base10_digits everywhere? if not, why not?

Reply:
Yeah, we can use base10_digits, I just thought this was more useful than it really was :p